include/internal/catch_config.hpp

```diff
@@ -50,7 +52,9 @@ namespace Catch {
         bool listTests;
         bool listTags;
         bool listReporters;
+        bool listTestNamesAndSources;
         bool listTestNamesOnly;
+		bool listTestSources;
```

Comment:
A) VS is a sneaky mistress
B) Whoops, I've been commiting to master when I have a pull request open